test/integration/scheduler/serving/flagz_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package serving
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"os"
+	"testing"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	featuregatetesting "k8s.io/component-base/featuregate/testing"
+	"k8s.io/component-base/zpages/features"
+	"k8s.io/klog/v2/ktesting"
+	kubeschedulertesting "k8s.io/kubernetes/cmd/kube-scheduler/app/testing"
+)
+
+func TestFlagzEndpoints(t *testing.T) {
```

Comment:
Can we reuse the `TestHealthEndpoints` and rename it to `TestEndpoints`, table driven test is more welcomed in kubernetes codebase, and I think it's feasible by just adding more cases there? WDYT?

Reply:
Updated, PTAL again. I fit the /flagz into the original test table.